spec/features/idv/inherited_proofing/inherited_proofing_cancel_spec.rb

```diff
@@ -0,0 +1,257 @@
+        end
+      end
+
+      context 'when clicking the "Exit Login.gov" button from the "Cancel" view' do
+        before do
+          click_link t('links.cancel')
+          expect(page).to have_current_path(idv_inherited_proofing_cancel_path(step: :get_started))
+        end
+
+        it 'redirects the user back to the service provider website' do
+          click_button t('idv.cancel.actions.exit', app_name: APP_NAME)
+          expect(page).to have_current_path(/\/auth\/result\?/)
+        end
+      end
+    end
+
+    context 'from the "How verifying your identify works" view, and clicking the "Cancel" link' do
+      before do
+        complete_idv_steps_up_to_inherited_proofing_how_verifying_step user
+      end
+
+      it 'should have current path equal to the How Verifying (agreement step) page' do
+        expect(page).to have_current_path(/inherited_proofing[?|\/].*agreement/)
+      end
+
+      context 'when clicking the "Start Over" button from the "Cancel" view' do
+        before do
+          click_link t('links.cancel')
+          expect(page).to have_current_path(idv_inherited_proofing_cancel_path(step: :agreement))
+        end
+
+        it 'redirects the user back to the start of the Inherited Proofing process' do
+          click_button t('inherited_proofing.cancel.actions.start_over')
+          expect(page).to have_current_path("#{idv_inherited_proofing_path}/get_started")
+        end
+      end
+
+      context 'when clicking the "No, keep going" button from the "Cancel" view' do
+        before do
+          click_link t('links.cancel')
+          expect(page).to have_current_path(idv_inherited_proofing_cancel_path(step: :agreement))
+        end
+
+        it 'redirects the user back to where the user left off in the Inherited Proofing process' do
+          click_button t('inherited_proofing.cancel.actions.keep_going')
+          expect(page).to have_current_path("#{idv_inherited_proofing_path}/agreement")
+        end
+      end
+
+      context 'when clicking the "Exit Login.gov" button from the "Cancel" view' do
+        before do
+          click_link t('links.cancel')
+          expect(page).to have_current_path(idv_inherited_proofing_cancel_path(step: :agreement))
+        end
+
+        it 'redirects the user back to the service provider website' do
+          click_button t('idv.cancel.actions.exit', app_name: APP_NAME)
+          expect(page).to have_current_path(/\/auth\/result\?/)
+        end
+      end
+    end
+
+    # Revisit. ATM cannot get around the :verify_wait step as it's an ajax-enabled step that an
+    # includes ActiveJob to move to the next step. Marking the necessary step as completed below,
+    # does not seem to do the trick for some reason. This has been manually verified in the
+    # meantime.
+    xcontext 'from the "We are retrieving your information..." view, and clicking "Cancel"' do
```

Comment:
Similar note as before about `xit` and skipped specs.

Reply:
Done.